### PR TITLE
[C-Api/Query] use caps property

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Use 'caps' property in appsrc, not caps-filter.

### CODE REVIEW

File: c/src/ml-api-service-query-client.c [hunk 1]
```diff
@@ -134,7 +134,7 @@ ml_service_query_create (ml_option_h option, ml_service_h * h)
   prop = g_string_free (tensor_query_client_prop, FALSE);
   description =
       g_strdup_printf
-      ("appsrc name=srcx ! %s ! tensor_query_client %s name=qcx ! tensor_sink name=sinkx async=false sync=false",
+      ("appsrc name=srcx caps=%s ! tensor_query_client %s name=qcx ! tensor_sink name=sinkx async=false sync=false",
```
Comment (gichan-jang) [inline c/src/ml-api-service-query-client.c:137]: Why do you change this?

Comment (jaeyun-jung) [reply c/src/ml-api-service-query-client.c:137]: I recommend using caps property when using appsrc, to clearly denote the caps from appsrc. caps-filter may not pass the caps to appsrc in some case.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM✔

**Pull request merged**